### PR TITLE
Remove deprecated functions from SecureRandom

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

File: lib/public/Security/ISecureRandom.php [hunk 1]
```diff
@@ -54,32 +54,6 @@
...
-	 *
-	 * Medium Strength should be used for most needs of a cryptographic nature.
-	 * They are strong enough to be used as keys and salts. However, they do
-	 * take some time and resources to generate, so they should not be over-used
-	 *
-	 * @return $this
-	 * @since 8.0.0
-	 * @deprecated 9.0.0 Use \OC\Security\SecureRandom::generate directly or random_bytes() / random_int()
-	 */
-	public function getMediumStrengthGenerator();
```
Comment (MorrisJobke) [inline lib/public/Security/ISecureRandom.php]: Used in here:

bildschirmfoto 2018-12-07 um 13 37 07

Comment (nickvergessen) [reply lib/public/Security/ISecureRandom.php]: Spreedme is dead

**Review by ChristophWurst — APPROVED** [review]

File: lib/public/Security/ISecureRandom.php (hunk 1 above)
Comment (rullzer) [reply lib/public/Security/ISecureRandom.php]: https://github.com/nextcloud/guests/pull/22
https://github.com/otetard/ownpad/pull/76

**Comment by rullzer** [comment]
Added to https://github.com/nextcloud/server/issues/12915

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Unit tests make 💥

**Pull request merged**